scripts/install.sh

```diff
@@ -0,0 +1,599 @@
+#!/bin/bash
```

Comment:
Cool.
I prefer to move this shell to a new director 'installer' and rename to 'install-datafuse.sh'.
install.sh will be the default script for installing the tool of datafuse cli, then we can install/upgrade the datafuse binaries via the cli